### PR TITLE
adds implemntation for logging commands,unknown command and urls

### PR DESCRIPTION
**Pull request by aabed**
I believe it will be more convenient to have all the data in a centralized place and display them in MHN
Also it will be part of cloning kippo-graph into MHN

### CODE REVIEW

File: kippo/dblog/hpfeeds.py [hunk 1]
```diff
@@ -203,9 +203,9 @@ def start(self, cfg):
 	# We have to return an unique ID
 	def createSession(self, peerIP, peerPort, hostIP, hostPort):
 		session = uuid.uuid4().hex
-		self.meta[session] = {'peerIP': peerIP, 'peerPort': peerPort, 
-		'hostIP': hostIP, 'hostPort': hostPort, 'loggedin': None,
-		'credentials':[], 'version': None, 'ttylog': None }
+		self.meta[session] = {'session':session,'peerIP': peerIP, 'peerPort': peerPort,
+        'hostIP': hostIP, 'hostPort': hostPort, 'loggedin': None,
```
Comment (jt6211) [inline kippo/dblog/hpfeeds.py]: Can you match the indentation style from the original file? I

**Comment by jt6211** [comment]
Other than the comment about the whitespace, these changes look great.  You will probably need to make some changes to mnemosyne to support this as well: https://github.com/threatstream/mnemosyne/blob/master/normalizer/modules/kippo_events.py#L22

**Comment by aabed** [comment]
I have fixed it, and of course I will add support into clio, but for now I will stick with them in the hpfeed collection instead of session 
Also I am thinking of adding startTime and endTime , will you be fine with that?

**Comment by jt6211** [comment]
I'm not sure if you only made these changes to your local copy, but it looks like they have not been pushed to GH yet.  

startTime and endTime seem reasonable to add.

**Pull request merged**